cmd/opampsupervisor/supervisor/supervisor.go

```diff
@@ -679,22 +671,10 @@ func (s *Supervisor) onOpampConnectionSettings(_ context.Context, settings *prot
 			return err
 		}
 	}
-	return s.waitForOpAMPConnection()
-}
-
-func (s *Supervisor) waitForOpAMPConnection() error {
-	// wait for the OpAMP client to connect to the server or timeout
-	select {
-	case <-s.connectedToOpAMPServer:
-		return nil
-	case <-time.After(10 * time.Second):
-		return errors.New("timed out waiting for the server to connect")
-	}
+	return nil
 }
 
-func (s *Supervisor) composeBootstrapConfig() ([]byte, error) {
-	var k = koanf.New("::")
-
+func (s *Supervisor) composeNoopPipeline() ([]byte, error) {
 	var cfg bytes.Buffer
 	err := s.bootstrapTemplate.Execute(&cfg, map[string]any{
```

Comment:
Can we rename the template as well if this is going to be a generic no-op pipeline rather than a bootstrapping-only pipeline?

Reply:
Renamed the template and this function to reference "noop" instead of bootstrap.